core/chains/evm/config/config.go

```diff
@@ -88,6 +88,9 @@ type ChainScopedOnlyConfig interface {
 	OCRObservationGracePeriod() time.Duration
 	OCRDatabaseTimeout() time.Duration
 
+	// OCR2 chain specific config
```

Comment:
Comments should start from the func/type name

Reply:
Not a comment for a function, but grouping of functions. Similar to line 85 in this file